src/client.rs

```diff
@@ -396,22 +405,7 @@ fn trace_server_loop(
             Ok(TraceCommand::Exit(_nanos, span_id)) => {
                 storage.exit_span(span_id);
             }
-            Ok(TraceCommand::Event(_nanos, thread_id, mut event, time)) => {
-                // Events are only valid if the trace_id flag is set
-                // Send trace specified the trace to send, so use that instead of the thread's
-                // current trace.
-                if let Some(send_trace_id) = event
-                    .remove("send_trace")
-                    .and_then(|t| t.parse::<u64>().ok())
-                {
-                    let send_vec = storage.drain_completed(send_trace_id, time);
-                    // Thread has ended this trace.  Until it enters a new span, it
-                    // is not in a trace.
-                    storage.remove_current_trace(send_trace_id);
-                    if !send_vec.is_empty() {
```

Comment:
Where did the client.send get moved to?  This is necessary to send the span list to datadog, correct?

Reply:
'client.send(spans)` was moved to where spans are closed.